packages/LUIS/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "luis-apis",
-  "version": "2.3.0",
+  "version": "2.4.0",
```

Comment:
We've already shipped 2.4.0, as part of this last cycle. 

Builds from master are "2.5.0-preview". 

Please update this to 2.5.0.

Reply:
fixed